Sources/Confidence/ConfidenceAppLifecycleProducer.swift

```diff
@@ -82,7 +91,7 @@ public class ConfidenceAppLifecycleProducer: ConfidenceEventProducer, Confidence
                 return
             }
             var context = self.currentProducedContext.value
-            context.updateValue(.init(boolean: isForeground), forKey: "is_foreground")
+            context.updateValue(.init(boolean: isForeground), forKey: Self.isForegroundContextKey)
```

Comment:
I think we should reconsider to have this context field at all.

Reply:
Can remove in new PR